steps/autify-upload/0.0.1/step.yml

```diff
@@ -0,0 +1,68 @@
+title: |
+  Autify Upload
+summary: |
+  Uploads your build to Autify.
+description: |-
+  Uploads you build `.app` to [Autify](https://autify.com/mobile) using Autify's API. You can upload the iOS build with the Step.
+
+  This Step, however, does NOT generate your build: to create a `.app` file, you need the right step. For example, `Xcode build for simulator` Step or any other Step can generate a build. Add task URL
+website: https://github.com/autifyhq/bitrise-steps-autify-upload
+source_code_url: https://github.com/autifyhq/bitrise-steps-autify-upload
+support_url: https://github.com/autifyhq/bitrise-steps-autify-upload/issues
+published_at: 2021-06-02T19:27:14.276215+09:00
+source:
+  git: https://github.com/autifyhq/bitrise-step-autify-upload.git
+  commit: 4e2d9cdd6a278cbb08aaf9b8276438ed6b6e9396
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
```

Comment:
is_requires_admin_user is deprecated and can be removed.

Reply:
ref:
- fixed at original repo. https://github.com/autifyhq/bitrise-step-autify-upload/pull/13